tests/fast-integration/test/fixtures/commerce-mock/index.js

```diff
@@ -265,16 +265,25 @@ async function connectCommerceMock(mockHost, tokenData) {
   }
 }
 
-async function ensureCommerceMockWithCompassTestFixture(client, appName, scenarioName, mockNamespace, targetNamespace, withCentralApplicationGateway=false) {
+async function ensureCommerceMockWithCompassTestFixture(client, appName, scenarioName, mockNamespace, targetNamespace, withCentralApplicationConnectivity = false) {
   const mockHost = await provisionCommerceMockResources(
-      `mp-${appName}`,
-      mockNamespace,
-      targetNamespace,
-      withCentralApplicationGateway ? prepareLastorderObjs('central-app-gateway-compass', `mp-${appName}`) : prepareLastorderObjs());
+    `mp-${appName}`,
+    mockNamespace,
+    targetNamespace,
+    withCentralApplicationConnectivity ? prepareLastorderObjs('central-app-gateway-compass', `mp-${appName}`) : prepareLastorderObjs());
   await retryPromise(() => connectMockCompass(client, appName, scenarioName, mockHost, targetNamespace), 10, 3000);
   await retryPromise(() => registerAllApis(mockHost), 10, 3000);
-  await waitForDeployment(`mp-${appName}-connectivity-validator`, "kyma-integration");
-  
+
+  if (withCentralApplicationConnectivity) {
+    await waitForDeployment('central-application-gateway', 'kyma-system');
+    await waitForDeployment('central-connectivity-validator', 'kyma-system');
+    await patchApplicationGateway('central-application-gateway', 'kyma-system');
+  } else {
+    await waitForDeployment(`mp-${appName}-connectivity-validator`, "kyma-integration");
+    await waitForDeployment(`mp-${appName}-application-gateway`, 'kyma-integration');
```

Comment:
I see the difference here. Previously we waited for ```await patchApplicationGateway(`${targetNamespace}-gateway`, targetNamespace);```. As far as I know, there is Gateway one-per-namespace used